### PR TITLE
Category products

### PR DESCRIPTION
**Pull request by m26unkwn**
1. Added the links to the category Section.
2. Click On the category section and check the products related to the category

### CODE REVIEW

File: src/components/CategoryPage/CategoryProducts.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import { useData } from "../../Context/stateManage/state-context";
+import ProductCard from "../Nuclie/ProductCard";
+const CategoryProducts = ()=>{
+
+    const {state:{products}} = useData()
+
+    const {categoryParam} = useParams();
+
+    const filterProducts = products.filter(product=> product.category === categoryParam);
+    console.log(filterProducts)
```
Comment (muskanbagrecha) [inline src/components/CategoryPage/CategoryProducts.jsx:11]: Remove console.log

File: src/App.js [hunk 2]
```diff
@@ -47,7 +48,11 @@ function App() {
         <Route path='/products' element={<ProductList />} />
         <Route path='/login' element={<Login />} />
         <Route path='/signup' element={<Signup />} />
-        <Route path='/product/:productId' element={<SingleProduct />} />
+        <Route path='/products/:productId' element={<SingleProduct />} />
+        <Route
+          path='/products/category/:categoryParam'
```
Comment (muskanbagrecha) [inline src/App.js:53]: This looks good.

File: src/components/CategoryPage/CategoryProducts.jsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import {useParams} from "react-router-dom";
```
Comment (muskanbagrecha) [inline src/components/CategoryPage/CategoryProducts.jsx:1]: Use prettier for formatting.

**Review by muskanbagrecha — APPROVED** [review]
Short PR. Working as expected. Good work.

**Review by Debarshi95 — APPROVED** [review]
All looks good. A suggestion, use a formatter like prettier to format your code. It will enhance code readability.

File: src/components/CategoryPage/CategoryProducts.jsx (hunk 1 above)
Comment (m26unkwn) [reply src/components/CategoryPage/CategoryProducts.jsx:11]: Definitely @muskanbagrecha

**Review by ritikaawasthy — COMMENTED** [review]
Suggestion:1. you could use an alert from you component library rather than using the default, it will look better.
2. You could increase the opacity of sol out card
code looks nice

**Pull request merged**